commit a1d757850e6e6fdad7ac2ebda8142f0e1419e7b9
Author: Friedemann Kleint <Friedemann.Kleint@digia.com>
Date:   Fri Dec 14 13:26:02 2012 +0100

    QMdiArea: Increase minimum size for Windows 8/Large fonts.
    
    Task-number: QTBUG-28611
    
    Change-Id: I115a5fa6726ac99c2bbff4ec65d3f8856b2530df
    Reviewed-by: Oliver Wolff <oliver.wolff@digia.com>
    (cherry picked from qtbase/3e3451f42989a0dbd2398da78ad84441c7898fd3)
    Reviewed-by: Janne Anttila <janne.anttila@digia.com>

diff --git a/tests/auto/qmdiarea/tst_qmdiarea.cpp b/tests/auto/qmdiarea/tst_qmdiarea.cpp
index e02eb5e..4bbcef1 100644
--- a/tests/auto/qmdiarea/tst_qmdiarea.cpp
+++ b/tests/auto/qmdiarea/tst_qmdiarea.cpp
@@ -1763,9 +1763,9 @@ void tst_QMdiArea::tileSubWindows()
     QTRY_COMPARE(workspace.viewport()->rect().size(), expectedViewportSize);
 
     // Not enough space for all sub-windows to be visible -> provide scroll bars.
-    workspace.resize(150, 150);
+    workspace.resize(160, 150);
     qApp->processEvents();
-    QTRY_COMPARE(workspace.size(), QSize(150, 150));
+    QTRY_COMPARE(workspace.size(), QSize(160, 150));
 
     // Horizontal scroll bar.
     QScrollBar *hBar = workspace.horizontalScrollBar();
